pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -1091,6 +1092,8 @@ func (cb *ClusterBuilder) buildUpstreamClusterTLSContext(opts *buildClusterOpts,
 				}
 			}
 		}
+		// Apply Ecdh Curves from MeshConfig
+		ApplyEcdhCurves(tlsContext, opts.mesh)
```

Comment:
```suggestion
		ApplyECDHCurves(tlsContext, opts.mesh)
```

nit - should be ECDH everywhere